bin/setup

```diff
@@ -2,10 +2,11 @@
 require "fileutils"
 
 # path to your application root.
-APP_ROOT = File.expand_path('..', __dir__)
+APP_ROOT = File.expand_path("..", __dir__)
+APP_NAME = "limber"
 
 def system!(*args)
-  system(*args) || abort("\n== Command #{args} failed ==")
+  system(*args, exception: true) || abort("\n== Command #{args} failed ==")
```

Comment:
I think the second part of the shortcut-or is redundant now because of the `exception: true` argument.
```
limber(dev)> system('ls nowhere')
ls: nowhere: No such file or directory
=> false
limber(dev)> system('ls nowhere', exception: true)
ls: nowhere: No such file or directory
(limber):2:in `system': Command failed with exit 1: ls (RuntimeError)
	from (limber):2:in `<main>'
```

Reply:
Maybe I should have left it as it was. I think the rails update suggested this change.

Reply:
reverted

Reply:
Thanks. Yes, if we want to preserve the error message and printing the command and args, we should keep it. Otherwise, it just prints a stack trace, which may not be helpful. Note: the wrapping method is used by bin/setup and bin/update scripts only for the development environment.